### ISSUE TITLE
Deploy section report

### ISSUE THREAD

**Issue by DavidRoy** [BiologicalRecordsCentre/UKBMS-online#3]
http://www.ukbms.org/mydata/section-plot

Actions needed.
1. report is not fully functioning.  After loading data (Monks Wood NNR), the drop down lists for selecting species do not work

2. deploy report to live site, as a sub-menu link under Reports menu heading #2

**Comment by Gary-van-Breda** [comment]
The section plot does not use the Year index values: these are used by the Year by Year index plot ( http://www.ukbms.org/mydata/year-plot ). This seems to be working OK for the Monks Wood NNR Index data. The section plot uses the actual counts data, which does not seem to be present for Monks Wood NNR. Shall we make it more obvious that the attempt to load data for the section plot has failed because there is no data to load?
Who should get to see the Reports - year plot and Section plot? Shall we put the File Upload, Visits Summary and Annual Summary also under the Reports menu item?

**Comment by DavidRoy** [comment]
Is working fine where there is data.  Agreed that we need a message if there is no data.

Reports - all should see them.
Agreed that there should be a Reports menu that contains:
Annual Summary
Visits Summary
Year-by-Year index plot
Section summary

Note that the 'File Upload' page should only be available to those with Admin rights.

**Comment by Gary-van-Breda** [comment]
There are problems with creating a submenu on Drupal 6 (current UKBMS Live), as the current menu system is not set up for submenus. It can be converted to a 'Nice menu' but that will require theming as it doesn't look the same. Given that we are working towards the D7 go-live, it doesn't seem worth expending the effort to create the submenu, so on the D6 system I have just added them to the normal menu
On the D7 site, I have set up the Reporting menu item, and I have put all 4 reporting elements under it, plus the "File Upload". In both D6 and D7 cases the reporting elements are all visible to "online recorder"s, and the "File Upload" to "regional coordinator"s.
As an aside, EBMS is consistent with this, with the same permissions required for access to the 5 items. It has a different menu structure, but it has its Reporting submenu as an entry in the Mydata menu.
